### ISSUE TITLE
[Question] How does the bot look for resources?

### ISSUE THREAD

**Issue by otomay** [AXeL-dev/Dindo-Bot#34]
Hello friend!

Would you help me? I don't understand anything about python hehehehe

What do you use to make the bot look for the resource?

I currently have a selfbot that searches for the entire image of the resource (I use it in mines). The biggest difficulty is that frequently (3/4 months) some images stop matching and I have to take prints again...

From the videos it looks like you use color matching and that the bot looks specifically in the area that you clicked on to store the data. That's right?

**Comment by otomay** [comment]
Oh, and my bot can also be considered a "pixel bot". Everything is done and read from the normal game screen.

**Comment by AXeL-dev** [comment]
Hello, i think that i already answered some of your questions so please refer to the following comments:
- https://github.com/Gamerium/Dindo-Bot/issues/16#issuecomment-613070702
- https://github.com/Gamerium/Dindo-Bot/issues/1#issuecomment-533335759
- https://github.com/Gamerium/Dindo-Bot/issues/7#issuecomment-536218843

More references:
- https://github.com/Gamerium/Dindo-Bot/issues?q=resource

Good luck on your bot :wink: .

**Comment by otomay** [comment]
Thanks for answering!

I'm reworking my bot this way, and it's much more efficient! I can't believe I used picture matching for so long hahahaha.

For the fights, I placed the picture frames of the fight participants on the left of the screen and pinned them, as well as hiding summons and dead characters from the timeline. To locate enemies I look for the blue color and to locate myself I look for the red color in that area where the blocks might be. So I cast the spells by clicking on the portraits instead of looking on the game :). I'm farming level 100 monsters with no problems with this technique.

I'm not very familiar with github issues best practices. Should I close? You helped me a lot!
